### ISSUE TITLE
Add additional loss functions

### ISSUE THREAD

**Issue by hendersontrent** [hendersontrent/ScratchNeuralNetwork.jl#6]
- [x] Mean Square Error loss
- [x] Cross-Entropy Loss

Can code up the Julia equivalents of them [here](https://machinelearningmastery.com/loss-and-loss-functions-for-training-deep-learning-neural-networks/).
